osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -172,7 +170,7 @@ protected override void UpdateStateTransforms(ArmedState state)
 
         public Drawable ProxiedLayer => ApproachCircle;
 
-        private class HitArea : Drawable, IKeyBindingHandler<OsuAction>
+        public class HitReceptor : Drawable, IKeyBindingHandler<OsuAction>
```

Comment:
why did you rename the class but not the variable?

Reply:
The rename prevents this:
```
public OsuAction? HitAction => HitArea.HitAction; // Compiler error (what is `HitArea`?)
public HitArea HitArea;
```

Open to suggestions.

Reply:
right... i guess it's ok for now then.